suite-native/feature-flags/src/useFeatureFlag.tsx

```diff
@@ -0,0 +1,24 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+    FeatureFlag,
+    FeatureFlagsRootState,
+    selectIsFeatureFlagEnabled,
+    toggleFeatureFlag,
+} from './featureFlagsSlice';
+
+type FeatureFlagHookReturnType = [boolean, () => void];
+
+export const useFeatureFlag = (key: FeatureFlag): FeatureFlagHookReturnType => {
+    const dispatch = useDispatch();
+
+    const isFeatureFlagEnabled = useSelector((state: FeatureFlagsRootState) =>
+        selectIsFeatureFlagEnabled(state, key),
+    );
+
+    const toggleIsFeatureFlagEnabled = () => {
```

Comment:
what about changing this to `setFeatureFlag` with a `boolean` parameter, since it is a general hook that will be used with all feature flags? I can imagine a scenario where I want to ensure that some feature flag is set e.g. to `false`. In that case, I could call `setFeatureFlag(false)` without prior checking of the current state as I would have to do with a `toggleFeatureFlag` function.

Reply:
Ok, if there should not be any other code that can change the feature flags, then it makes sense like this. 👍